Targets/Apps/watchOS-App.xcconfig

```diff
@@ -0,0 +1,14 @@
+//
+// watchOS-App.xcconfig
+//
+// Copyright (c) 2016 Twitter. All rights reserved.
+//
+
+#include "AppTarget.xcconfig"
+
+ARCHS = $(ARCHS_STANDARD)
```

Comment:
Similar note here, they are both included in `Target.xcconfig`, but `Architectures.xcconfig` doesn't specify a value for `ARCHES`. Each of the app xcconfigs specifies a value for this setting (they are not all the same)